config/default.yaml

```diff
@@ -365,6 +365,10 @@ instance:
 #    - 18 # Food
 
   default_client_route: '/videos/trending'
+  default_trending_route: '/videos/most-viewed'
+  pages:
```

Comment:
```
algorithms:
  enabled: 
    - 'views-interval' # Default one, used by PeerTube since the beginning
    - 'reddit-hot' # Takes into account...
  default: 'views-interval'
```

This way we can easily add algorithms in the future. Then, you just build the default trending route yourself depending on the default algorithm. You could just use a query parameter in the client to define the chosen algo: `/videos/trending?algorithm=hot`